### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by uibhanu5**
Profile UI

### CODE REVIEW

File: src/app/views/admin-views/profile-module/components/profile-page/profile.component.html [hunk 1]
```diff
@@ -45,11 +44,14 @@
...
           <div class="h-px-90">
-            <ion-item >
-              <ion-label>Gender</ion-label >
-              <ion-select placeholder="Select Gender" class="drop-down" formControlName='gender_id'>
-                <ion-select-option value=1>Female</ion-select-option>
-                <ion-select-option value=2>Male</ion-select-option>
+            <ion-item>
+              <ion-label color="medium">Gender</ion-label>
+              <ion-select class="ion-text-end drop-down" name="gender_id" formControlName="gender_id">
+                <!-- <ion-select-option *ngFor="let item of genderListData" value="{{item.gender_id}}">
```
Comment (Nithyam1330) [inline src/app/views/admin-views/profile-module/components/profile-page/profile.component.html]: Remove commented code

File: src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts [hunk 2]
```diff
@@ -36,6 +39,13 @@ export class ProfileComponent  extends BaseClass implements OnInit{
     }
 
   ngOnInit() {
+    this.commonRequestService.request(RequestEnums.GET_GENDER_TYPES).subscribe(response=>{
```
Comment (Nithyam1330) [inline src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts]: Write in seperate function and call it inside OnInit

File: src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts [hunk 3]
```diff
@@ -61,7 +71,10 @@ export class ProfileComponent  extends BaseClass implements OnInit{
   }
 
   onSubmit(){
-    console.log(this.profileForm.value)
+    console.log(this.profileForm.value);
```
Comment (Nithyam1330) [inline src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts:77]: Remove console logs

**Review by Nithyam1330 — APPROVED** [review]

File: src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts (hunk 2 above)
Comment (uibhanu5) [reply src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts]: Done

File: src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts (hunk 3 above)
Comment (uibhanu5) [reply src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts:77]: Done

File: src/app/views/admin-views/profile-module/components/profile-page/profile.component.html (hunk 1 above)
Comment (uibhanu5) [reply src/app/views/admin-views/profile-module/components/profile-page/profile.component.html]: Done

**Pull request merged**